requirements.txt

```diff
@@ -0,0 +1,8 @@
+arrow
+bleach
+fedmsg[all]
```

Comment:
Well, I ran into a lot of missing dependencies because this package imports [fedmsg.consumers](https://github.com/fedora-infra/fmn.consumer/blob/develop/fmn/consumer/consumer.py#L9) which [imports psutils](https://github.com/fedora-infra/fedmsg/blob/develop/fedmsg/consumers/__init__.py) and whose modules depend on things like [pygments](https://github.com/fedora-infra/fedmsg/blob/develop/fedmsg/consumers/ircbot.py#L34). I actually noticed that consumers imports from ``fedmsg.crypto`` as well so I'm sort of leaning back in the direction of using ``fedmesg[all]``, but it seems to fail gracefully if the packages aren't present (although I assume it doesn't check signatures when that happens so maybe that's not good?).

Since the fedmsg [setup.py](https://github.com/fedora-infra/fedmsg/blob/develop/setup.py#L75) has an ``extras_require`` helpfully defined I thought it best to use it.